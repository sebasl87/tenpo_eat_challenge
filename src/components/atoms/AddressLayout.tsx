import React from 'react'
import styled from 'styled-components/native';

export interface IRestaurantsLayout {
    children: React.ReactNode;
}
const Layout = styled.View`    
    background: #D4F9F5;
    padding: 20px 0px 18px 0px;
    border-top-left-radius: 28px;
    border-top-right-radius: 28px;
    align-items: center;
`
const AddressLayout: React.FC<IRestaurantsLayout> = ({ children }) => {
    return (
        <Layout>
            {children}
        </Layout>
    );
}

export default AddressLayout;