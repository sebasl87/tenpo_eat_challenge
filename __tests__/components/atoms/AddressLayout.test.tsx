import "react-native";
import React from "react";
import renderer from "react-test-renderer";
import AddressLayout from "../../../src/components/atoms/AddressLayout";


describe("AddressLayout component", () => {
    it("should render successfully with AddressLayout", () => {
        const titleSection = renderer
            .create(<AddressLayout children="AddressLayout_mock" />)
            .toJSON();

        expect(titleSection).toMatchSnapshot();
    });
});
