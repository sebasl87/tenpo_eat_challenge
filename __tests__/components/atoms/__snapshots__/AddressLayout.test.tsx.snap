// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddressLayout component should render successfully with AddressLayout 1`] = `
<View
  style={
    [
      {
        "alignItems": "center",
        "backgroundColor": "#D4F9F5",
        "borderTopLeftRadius": 28,
        "borderTopRightRadius": 28,
        "paddingBottom": 18,
        "paddingLeft": 0,
        "paddingRight": 0,
        "paddingTop": 20,
      },
    ]
  }
>
  AddressLayout_mock
</View>
`;
